src/benchmarks/notcoin/results_code_size.json

```
@@ -0,0 +1,24 @@
+{
+  "results": [
+    {
+      "label": "FunC",
+      "size": {
+        "minter cells": "38",
+        "minter bits": "15898",
+        "wallet cells": "16",
+        "wallet bits": "7384"
+      },
+      "pr": "https://github.com/OpenBuilders/notcoin-contract"
+    },
+    {
+      "label": "1.6.2",
+      "pr": "https://github.com/tact-lang/tact/pull/2229",
```

```suggestion
      "pr": "https://github.com/tact-lang/tact/pull/2329",
```

The #2229 didn't feature notcoin

---

Changed